DTXManiaプロジェクト/コード/スコア、曲/CSong管理.cs

```diff
@@ -761,6 +762,13 @@ private Cスコア tSongsDBからスコアを１つ読み込む( BinaryReader br
 			cスコア.譜面情報.Bpm = br.ReadDouble();
 			cスコア.譜面情報.Duration = br.ReadInt32();
             cスコア.譜面情報.strBGMファイル名 = br.ReadString();
+            cスコア.譜面情報.SongVol = br.ReadInt32();
+		    var hasSongIntegratedLoudness = br.ReadBoolean();
```

Comment:
SONGVOL support requires that the song db version be incremented. Song Integrated Loudness and Song Peak Loudness have been added now so that we don't have to break the db file format again when I complete my next batch of work.